drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -3970,7 +3974,11 @@ void RasterizerSceneGLES3::render_scene(const Transform& p_cam_transform,const C
 		_render_mrts(env,p_cam_projection);
 	}
 
+#ifndef IPHONE_ENABLED
+	// glPolygonMode has mostly been deprecated on OpenGL ES, basically GL_FILL is the only mode still available.
+	// For some really strange reason GL_FILL is no longer defined within Apples ES3/gl.h file making glPolygonMode defunct
 	glPolygonMode(GL_FRONT_AND_BACK,GL_FILL);
+#endif
```

Comment:
I agree, there is nowhere in Godot where it is ever change (and from what I've read there is no option on ES or 3+).